sklearn/cluster/tests/test_k_means.py

```diff
@@ -31,6 +32,7 @@
 from sklearn.datasets import make_blobs
 from io import StringIO
 
+pytestmark = pytest.mark.filterwarnings("ignore::FutureWarning")  # TODO(1.3): Remove
```

Comment:
I do not think we should filter out all the warnings. I suggest updating `KMeans` everywhere here and pass auto.

Reply:
I updated the filter to be more specific -- I think this should be fine, right?
https://github.com/scikit-learn/scikit-learn/blob/4b6be5028b692c015b0e1818ae39a3c3162f04d3/sklearn/cluster/tests/test_k_means.py#L35-L39